restocking/src/test/scala/dev/atedeg/mdm/restocking/Tests.scala

```diff
@@ -0,0 +1,48 @@
+package dev.atedeg.mdm.restocking
+
+import cats.data.NonEmptyList
+import org.scalatest.GivenWhenThen
+import org.scalatest.featurespec.AnyFeatureSpec
+import org.scalatest.matchers.should.Matchers
+
+import dev.atedeg.mdm.products.{ Ingredient, Product }
+import dev.atedeg.mdm.products.Ingredient.{ Cream, Milk, Probiotics, Rennet, Salt }
+import dev.atedeg.mdm.utils.given
+
+trait Mocks {
+  private val milk = QuintalsOfIngredient(WeightInQuintals(5.5), Milk)
+  private val cream = QuintalsOfIngredient(WeightInQuintals(3.0), Cream)
+  private val rennet = QuintalsOfIngredient(WeightInQuintals(2.0), Rennet)
+  private val salt = QuintalsOfIngredient(WeightInQuintals(3.0), Salt)
+  private val probiotics = QuintalsOfIngredient(WeightInQuintals(0.1), Probiotics)
+
+  val ingredients: NonEmptyList[QuintalsOfIngredient] = NonEmptyList.of(milk, cream, rennet, salt, probiotics)
+
+}
```

Comment:
Since we are using scala 3 you could drop the `{}` and use `:`
The same applies for the `ActionTest` class

Reply:
Got it. `ActionTest` is of @nicolasfara's bc, should I fix it or should he?

Reply:
Also, I noticed neither production nor stocking is using scala 3 syntax in tests